Components/category.css

```diff
@@ -0,0 +1,28 @@
+.category-cards{
+    display: flex;
+    justify-content: space-evenly;
+    flex-wrap: wrap;
+}
+.category-container h2{
+    text-align: center;
+    margin: 0;
```

Comment:
define margin 0 in reset CSS child elements inherits from it.

Reply:
Thanks Done...